src/components/CreateVmWizard/steps/SummaryReview.js

```diff
@@ -0,0 +1,344 @@
+}
+
+const ReviewStorage = ({ id, storageDomains, storage }) => {
+  return <Item id={`${id}-storage`} label={msg.createVmWizardStepTitleStorage()}>
+    { storage.length === 0 && <div>{msg.createVmStorageEmptyInfo()}</div> }
+
+    { storage.length > 0 &&
+      storage.map(disk =>
+        <div key={disk.id}>
+          <div className={style['review-entity-info']}>
+            <DiskNameWithLabels id={id} disk={disk} />
+          </div>
+          <div className={style['review-entity-info']}>
+            { userFormatOfBytes(disk.size, 'B').str }
+          </div>
+          <div className={style['review-entity-info']}>
+            { storageDomains.getIn([ disk.storageDomainId, 'name' ], msg.createVmStorageUnknownStorageDomain()) }
+          </div>
+          <div className={style['review-entity-info']}>
+            {
+              disk.format === 'cow' ? msg.diskEditorFormatOptionCow()
+                : disk.format === 'raw' ? msg.diskEditorFormatOptionRaw()
+                  : disk.format
+            }
+          </div>
+          {/* TODO: Include disk interface? */}
+        </div>
+      )
+    }
+  </Item>
+}
+ReviewStorage.propTypes = {
+  id: PropTypes.string,
+  storageDomains: PropTypes.object,
+  storage: PropTypes.arrayOf(PropTypes.shape(STORAGE_SHAPE)).isRequired,
+}
+
+const ReviewAdvanced = ({ id, operatingSystems, basic }) => {
+  const hasAdvanced = basic.startOnCreation || basic.cloudInitEnabled
+  const vmOS = operatingSystems.get(basic.operatingSystemId)
+
+  if (!hasAdvanced) {
+    return null
+  }
+
+  return <Item id={`${id}-advanced-options`} label={'Advanced'}>
+    { basic.startOnCreation && <div id={`${id}-startOnCreation`}>{msg.startVmOnCreation()}</div> }
+    { basic.cloudInitEnabled && !vmOS.get('isWindows') &&
+      <React.Fragment>
+        <div>{msg.createVmWizardReviewAdvancedCloudInit()}</div>
+        <div className={style['review-subsection']}>
+          { basic.initHostname &&
+            <Item id={`${id}-cloud-init-hostname`} label={msg.hostName()}>{ basic.initHostname }</Item>
+          }
+          { basic.initSshKeys &&
+            <Item id={`${id}-cloud-init-sshkey`} label={msg.sshAuthorizedKeys()}>{ basic.initSshKeys }</Item>
+          }
+        </div>
+      </React.Fragment>
+    }
+    { basic.cloudInitEnabled && vmOS.get('isWindows') &&
+      <React.Fragment>
+        <div>{msg.createVmWizardReviewAdvancedSysprep()}</div>
+        <div className={style['review-subsection']}>
+          { basic.initHostname &&
+            <Item id={`${id}-sysprep-hostname`} label={msg.hostName()}>{ basic.initHostname }</Item>
+          }
+          { basic.initTimezone &&
+            <Item id={`${id}-sysprep-tz`} label={msg.sysPrepTimezone()}>{ basic.initTimezone }</Item>
+          }
+          { basic.initAdminPassword &&
+            <Item id={`${id}-sysprep-admin-pwd`} label={msg.sysPrepAdministratorPassword()}>
+              { '*'.repeat(basic.initAdminPassword.length) }
```

Comment:
Maybe here will be better to use some default length, for example 6 asterisks? Because it is not very secure to show length of password.

Reply:
They entered it themselves 3 steps ago, so not sure how secure it needs to be.  Plus it'll just get logged in the comms anyway.

For sake of argument, I'll just make it a static 6 asterisks.